### PR TITLE
Add ERC: Equity Shares Book-Entry System

### PR DESCRIPTION
**Pull request by paul-lee-attorney**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: ERCS/erc-7598.md [hunk 1]
```diff
@@ -0,0 +1,2756 @@
+---
+eip: 7598
```
Comment (abcoathup) [inline ERCS/erc-7598.md:2]:
```suggestion
eip: 7596
```
This PR appears to be a duplicate of #204.  Please close this duplicate.  
FYI, EIP/RIP/ERC numbers are assigned by editors and associates.

Comment (paul-lee-attorney) [reply ERCS/erc-7598.md:2]: Well understood. Many thanks. 
I will close this PR and update the ERC number as 7596.  

Regards.
Paul

Comment (paul-lee-attorney) [reply ERCS/erc-7598.md]: eip number in the head, as well as the file name have been changed as per the new assigned number.  
Please review whether it's compliance with the rules. 

Many thanks.
Paul

Comment (paul-lee-attorney) [reply ERCS/erc-7598.md]: Hi Andrew,  
Could you please approve this PR so as to incorporated it into the Repo?

File: ERCS/erc-7596.md [hunk 2]
```diff
@@ -0,0 +1,2756 @@
...
+discussions-to: https://ethereum-magicians.org/t/add-erc-equity-shares-book-entry-system/18155
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-01-15
+---
+
+## Abstract
+
+This ERC describes the details of the Equity Shares Book-Entry System (the "**System**"), a proposed protocol and ABI definition that can automatically manage the book-entry records of a company with respect to its equity shares and corporate governance behaviors, in accordance with the consitutional document thereof, so as to enable investors, shareholders, managers, and other stakeholders can directly engage in almost all equity share transactions and corporate governance activities under the automatic control of smart contracts, including share subscription, share transfer, share pledge, share consideration payment, share listing, constitution updates, contracts signing, motion submission, voting, candidates nomination, taking office and resignation etc..
```
Comment (SamWilsn) [inline ERCS/erc-7596.md:15]: This is an extremely long sentence. While not strictly required, I'd very much recommend breaking it up to make reading easier :rofl:

File: ERCS/erc-7596.md [hunk 3]
```diff
@@ -0,0 +1,2756 @@
...
+
+Under the automatic control of smart contracts, many “claim rights” that require cooperation of obligors are transformed into "disposal rights" that can be unilaterally exercised by right holders. In this way, the possibility of defaults will be completely eliminated. So that, the System may fundamentally solve the corporate governance problems such as **"Insider Control"** and **"Misleading Statements"** that has plagued the capital markets for many years, and completely realize:
+1. right holders may **directly exercise** their rights; 
+2. obligors have **no chance** to default; and
+3. **real-time** information disclosure.
+
+By booking the equity shares on EVM blockchain, investors may use cryptocurrency or Central Bank Digital Currencies to pay for share consideration, which means effectively connecting the company’ equity shares to the trillion-dollar crypto markets, so that the companies may have an opportunity to have their equity value reevaluated and reconfirmed with the robust support of the huge liquidity.
```
Comment (SamWilsn) [inline ERCS/erc-7596.md:22]: Overall, not a bad abstract. I think it could do with a bit of condensing, and a bit more technical details.

File: ERCS/erc-7596.md [hunk 4]
```diff
@@ -0,0 +1,2756 @@
...
+2. All legal and commercial rights of shareholders (or members), conditions and procedures of exercising the shareholders' rights, hiearachy structure of the company as well as corporate governance rules concerned, all those aspects of a company **SHALL** be set out and governed by provisions of the Constitutional documents (i.e. Shareholders Agreement);
+3. Register of Shares privately managed by the company shall be the fundamental and decisive records to create and verify the legal identity and economic rights of shareholders;
+4. Shareholders (or members), as wealth owner of the equity shares, **SHALL** have rights to transfer, pledge, create options or other legal encumbrances upon the equity shares they owned;
+5. Shareholders (or members), as beneficial owner of the company, **SHALL** have rights to engage in corporate governance activites, including but not limited to proposing motions, voting on motions, nominating candidates (of directors or managers), convening meetings, executing motions etc.;
+6. Agreements **SHALL** have binding forces upon the signing parties; and
+7. other legal and commercial principles that can hardly be exaustively specified.
```
Comment (SamWilsn) [inline ERCS/erc-7596.md:2739]: The uppercase keywords (eg. "SHALL") create requirements on implementations, and so should be in the specification section. If you're just explaining requirements that have already been created above, then don't use the uppercase keywords.

File: ERCS/erc-7596.md [hunk 5]
```diff
@@ -0,0 +1,2756 @@
...
+
+## Backwards Compatibility
+
+This is the first edition of this ERC governing the on-chain behaviors of members and stakeholders of a company with respect to equity share transaction and corporate governance.  However, the later updated versions **SHALL** keep compatible to the current edition in views of legal, commercial and technical consistance.
```
Comment (SamWilsn) [inline ERCS/erc-7596.md:2743]: You can't really create requirements on future proposals. People can (and will) do whatever they want.

**Review by SamWilsn — COMMENTED** [review]
This is a _huge_ proposal. It will be quite difficult to get sufficient technical review from the community if it stays grouped up like this. Are there any natural points where it could be split into smaller documents?

Size aside, at first glance it seems _very_ specific to a particular organizational hierarchy. I feel like I need to ask: who are the different parties you expect to need to interoperate? Proposals here really only shine when there are many independent groups that need to interoperate. Aside from members of the company, who do you expect to use all these functions? Is this document better suited to a Solidity library or template that orgs can choose to deploy?

Consider ERC-20. It's a great example of something that should be in an ERC. All tokens want to support all wallets, and all wallets want to support all tokens. Image if every wallet had to build bespoke code to handle each token; it'd be impossible. With this proposal, it seems like you'd have a single frontend for each org.?

**Comment by paul-lee-attorney** [comment]
> This is a _huge_ proposal. It will be quite difficult to get sufficient technical review from the community if it stays grouped up like this. Are there any natural points where it could be split into smaller documents?
> 
Thank you very much for your attention and comments.  Yes. It is a very complicated and Huge topic.  Actually, I'm trying to propose a dynamic model of corporate governance and equity transaction, which may generally compatible to all kinds of companies or corporates.  It can be split into small topics like Register of Shares, or, structure and method interfaces of Shares etc.  While, considering the legal relationship among the records of corporate governance, there needs to be a general specification or standard to give a general picture to define the controlling relationship among the different registers.  That is why I put all books/registers togethet into this ERC proposal.


> Size aside, at first glance it seems _very_ specific to a particular organizational hierarchy. I feel like I need to ask: who are the different parties you expect to need to interoperate? Proposals here really only shine when there are many independent groups that need to interoperate. Aside from members of the company, who do you expect to use all these functions? Is this document better suited to a Solidity library or template that orgs can choose to deploy?

All stakeholders of a company, and, subsidiaries of a company, can conduct legal acts directly on chain.  This the scenario I imagine the system/standard may implement. It will be a set of templates to be selected by the companies based on their specific governance design.  E.g. if they are members managed companies, they can only deploy Register of Members and General Meeting Minutes these two smart contracts, while, if they decided to set up a Board, they may add Register of Directors and Board Meeting Minutes into their system as well.  

Pls have a look at my project website for white paper and source code at https://comboox.vercel.app

> 
> Consider ERC-20. It's a great example of something that should be in an ERC. All tokens want to support all wallets, and all wallets want to support all tokens. Image if every wallet had to build bespoke code to handle each token; it'd be impossible. With this proposal, it seems like you'd have a single frontend for each org.?

Yes. I'm trying to set up a general compatible model or standards to fit all kinds of corporate governance.

File: ERCS/erc-7596.md (hunk 2 above)
Comment (paul-lee-attorney) [reply ERCS/erc-7596.md:15]: Thank you for your suggestion. I will try to simplify the wording with plain language.
